xarray_user_forum_2021/Xarray_User_Summit.ipynb

```diff
@@ -0,0 +1,9690 @@
+{
```

Comment:
<p>I think it should be "these numbers are"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_misc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>